lib/encoder.py

```diff
@@ -301,6 +301,19 @@ def RemoveValue(self, name):
     del new_set.values[name]
     return new_set
 
+  def RemoveValue(self, name):
```

Comment:
Yes, but git rebase did something strange to that commit, so that "git rebase master" now doesn't do what I expected it to do.
Committing.